nextstrain/auspice
# Add screenshot testing

### Description of proposed changes    
This PR adds `jest-image-snapshot` and `jest.retryTimes()` to do basic screenshot diffing for website's functionality

### Related issue(s)  
Related to #917 

### Testing
What steps should be taken to test the changes you've proposed?  
If you added or changed behavior in the codebase, did you update the tests, or do you need help with this?

Please help test this PR locally by:

1. Make sure you have the default `zika.json` data via: `curl http://data.nextstrain.org/zika.json --compressed -o data/zika.json`

2. Run `npm run dev`

3. In another terminal tab run: `npm run integration-test` and all snapshot tests should pass:

<img width="788" alt="Screen Shot 2020-04-15 at 9 09 27 PM" src="https://user-images.githubusercontent.com/6309723/79413671-6f0c8900-7f5d-11ea-8a9d-e2a50b218523.png">

### Thank you for contributing to Nextstrain!
